src/.kateconfig

```diff
@@ -0,0 +1 @@
+kate: indent-width 3;
```

Comment:
I think you can use .editorconfig instead of adding this file. Looks like support was added https://github.com/editorconfig/editorconfig.github.com/issues/78.

Reply:
If you prefer to rely on that, I understand, and I'm willing to drop that commit, but FWIW that's not even in latest Fedora (or at least, it doesn't appear to be *working* for me). Maybe in a few years that will work for most users, but right now it probably only works for users that have built katepart from bleeding-edge sources.